torch/backends/xeon/run_cpu.py

```diff
@@ -564,7 +564,7 @@ def _add_multi_instance_params(parser):
                        help="Disable numactl")
     group.add_argument("--core_list", metavar="\b", default=None, type=str,
                        help="Specify the core list as \"core_id, core_id, ....\", otherwise, all the cores will be used.")
-    group.add_argument("--log_path", metavar="\b", default="logs", type=str,
+    group.add_argument("--log_path", metavar="\b", default="", type=str,
```

Comment:
Why do we want to change the default behavior here? Also, is changing the [torch/distributed/elastic/multiprocessing/__init__.py](https://github.com/pytorch/pytorch/pull/88477/files#diff-b3b139756008a8adb2e3cf5c927cf16e2c4b132e362370fe4dc9e50e8991f4d2) necessary?